### PR TITLE
Create XML

### PR DESCRIPTION
**Pull request by nvs266**
### Summary:
Create XML, add drawable

### Effect:
- None

### Link Issue:
https://github.com/framgia/fgoal-android/issues/29

### CODE REVIEW

File: app/src/main/java/com/fs_sournary/framgia/fgoal/screen/okrs/OKRPagerAdapter.java [hunk 1]
```diff
@@ -27,7 +25,7 @@ public Fragment getItem(int position) {
             case TabType.CHARTS:
                 return new ChartFragment();
             case TabType.OKRS:
-                return new Fragment();
+                return new TabOKRFragment();
```
Comment (DoanVanToan) [inline app/src/main/java/com/fs_sournary/framgia/fgoal/screen/okrs/OKRPagerAdapter.java]: rename to OKRFragment

File: app/src/main/res/layout/fragment_tab_okr.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            android:id="@+id/text_completion"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/dp_2"
+            android:text="@string/msg_completion"
+            android:textAppearance="@style/TextAppearance.AppCompat.Menu"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_tab_okr.xml]: remove bl

File: app/src/main/java/com/fs_sournary/framgia/fgoal/screen/okrs/OKRPagerAdapter.java (hunk 1 above)
Comment (nvs266) [reply app/src/main/java/com/fs_sournary/framgia/fgoal/screen/okrs/OKRPagerAdapter.java]: (resolved)

File: app/src/main/res/layout/fragment_tab_okr.xml (hunk 2 above)
Comment (nvs266) [reply app/src/main/res/layout/fragment_tab_okr.xml]: (resolved)

**Pull request merged**